apps/test/unit/templates/studioHomepages/TeacherHomepageTest.jsx

```diff
@@ -172,6 +204,18 @@ describe('TeacherHomepage', () => {
     expect(screen.getByText('My Dashboard')).toBeInTheDocument;
   });
 
+  it('shows the FAB button', async () => {
```

Comment:
The FAB button at the ATM machine? :-)

```suggestion
  it('shows the FAB', async () => {
```